README.md

```diff
@@ -11,7 +11,7 @@
 >
 > This readme and related documentation are Work in Progress.
 
-Chaos Mesh® is a cloud-native Chaos Engineering platform that orchestrates chaos on Kubernetes environments. At the current stage, it has the following components:
+Chaos Mesh® is a [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) hosted project. It is a cloud-native Chaos Engineering platform that orchestrates chaos on Kubernetes environments. At the current stage, it has the following components:
```

Comment:
Chaos Mesh® is a cloud-native Chaos Engineering platform that orchestrates chaos experiments on Kubernetes environments. It is a [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) sandbox project. At the current stage, Chaos Mesh has the following components:

Reply:
@BellaXiang